test/components/structures/MatrixChat-test.tsx

```diff
@@ -676,7 +683,6 @@ describe("<MatrixChat />", () => {
                         it("should warn when room is not public", async () => {
                             jest.spyOn(room.currentState, "getStateEvents").mockReturnValue(inviteJoinRule);
                             dispatchAction();
-                            await screen.findByRole("dialog");
```

Comment:
I'm not sure removing this is expected, likely will make the test racey

Reply:
ah yes, in fact it just made it fail outright